Tasks/AzureVmssDeployment/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 7
+        "Patch": 8
```

Comment:
I hope that as you mentioned, you will port these changes to releases/m123 and master as well and ensure that the versions there are greater than this version.

Reply:
yes i will do that